modules/machineset-manually-scaling.adoc

```diff
@@ -39,7 +39,7 @@ $ oc get machine -n openshift-machine-api
 +
 [source,terminal]
 ----
-$ oc annotate machine/<machine_name> -n openshift-machine-api machine.openshift.io/cluster-api-delete-machine="true"
+$ oc annotate machine/<machine_name> -n openshift-machine-api machine.openshift.io/delete-machine="true"
```

Comment:
In the other file, you have delete-machine="yes" and this file has delete-machine="true" - I don't know if this is an issue but thought I would point it out. Also this version has a slash before <machine_name> where the other version does not - again, not sure if significant but am pointing it out. (Realizing that neither of these are things that you changed in this PR, but I thought I'd note them just in case there's an issue.)

Reply:
Both good questions. The file that differs is outside the scope of my knowledge - @johnwilkins or @elmiko do you have any input about this difference?

Reply:
Edit: I should have tagged @bfournie and @lshilin-r in this PR, and they may be able to weigh in on the baremetal vs. "normal" difference here.

Reply:
fwiw, the "value" for the annotation key doesn't matter. it could be `true`, `yes`, or even `yes-with-syrup-on-top`. the code will  look for the presence of the annotation key with any value, but it should have a value.

Reply:
It can be both `true` and `yes`.  And it can be mentioned by Note if you like